app/auth.py

```
@@ -0,0 +1,122 @@
+    credentials: Annotated[HTTPAuthorizationCredentials, Depends(security)],
+    db: Annotated[Session, Depends(get_db)],
+) -> User:
+    """Get current authenticated user from JWT token."""
+    token = credentials.credentials
+
+    payload = verify_token(token)
+    username = payload.get("sub")
+    jti = payload.get("jti")
+    token_type = payload.get("type")
+
+    if not username or not isinstance(username, str) or token_type != "access":
```

Ruff (0.14.13)

Possible hardcoded password assigned to: "token_type"

(S105)